### ISSUE TITLE
 fullpage.js not scrolling after the 6th sections when the screen width is equal to or smaller than 980px

### ISSUE THREAD

**Issue by JasonnnC** [alvarotrigo/fullPage.js#4525]
Hello, I've been device-testing one of the pages on my website. I ran into an issue with fullpage.js not scrolling after a certain number of sections when on a tablet or mobile. The problem only appears when the screen width is equal to or smaller than 980px. After the 6th section, the scroll gets stuck like it's at the bottom of the page. I can scroll back up, but I cannot scroll any further down. This means there are 3 extra sections which visitors cannot currently see on a tablet and mobile. Any advice or help would be greatly appreciated. Thanks.

**Comment by alvarotrigo** [comment]
Can you please provide an isolated reproduction in [jsfiddle](https://jsfiddle.net/alvarotrigo/ea17skjr/) or [codepen](https://codepen.io/alvarotrigo/pen/qqabrp), with no CSS or JS files external to fullPage.js and the minimum amount of HTML code? Use empty sections unless strictly necessary for the reproduction.

I'd say its a HTML markup issue of some kind, check the JS console to see if you see any errors.

**Comment by JasonnnC** [comment]
@alvarotrigo I cannot replicate the issue or find any errors within the JS Console. Could I send you a private email with my site's URL so you can see the problem for yourself there? The site is still in staging so I don't want to publically share it for security reasons.

**Comment by alvarotrigo** [comment]
> Could I send you a private email with my site's URL so you can see the problem for yourself there?

Sure, as long as you get a Business License from the <a href="https://alvarotrigo.com/fullPage/pricing/"  target="_blank">fullPage pricing page</a> you can get access to premium support. 

Otherwise, you can try to reproduce your issue in a codepen and I'll help you out here.

**Comment by JasonnnC** [comment]
@alvarotrigo sorry for the late response. I managed to fixIt looksissue. It looks like it was more straightforward than I realised. The issue was that there were two columns with 100% height, so when the screen dropped to tablet size these were positioned stacked and took up the section they were on as well as the section below. As soon as I changed the size and made both fit on the one section it fixed the scroll.
